README.md

```diff
@@ -15,10 +15,19 @@ The API is used by other apps to notify users in the web UI and sync clients abo
 
 ## Developers
 
-### Build the notifications app
+### Install and enable the notifications app
 
-To set up this app for development, you need to run `make dev-setup` from within the app´s root folder. If anytime later you need to rebuild the javascript files the quicker `make build-js` is enough.
+- Clone this app into the "apps" folder of your nextcloud instance.
+```
+git clone https://github.com/nextcloud/notifications.git
+```
 
+- Execute `make dev-setup` from within the app´s root folder (If anytime
+ later you need to rebuild the javascript files the quicker `make build-js` is enough).
```

Comment:
This is only required when developing. The repository contains ready compiled JS assets and simply enabling the app after cloning should be enough.
In fact this could even cause issues (when done with a wrong node/npm version), so I would prefer to skip this part here.